### PR TITLE
Add extension: Bible

### PR DESCRIPTION
**Pull request by josmithua**
## Description

The Bible extension is aimed at searching for and displaying bible content. Currently you can search by bible reference. The extension uses biblegateway.com to perform searches.

## Screencast

[Link to screencast](https://drive.protonmail.com/urls/PMB7Y6530M#8yOu79ztI1s7)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/bible/package.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "bible",
+  "title": "Bible",
+  "description": "An extension for all things bible-related.",
```
Comment (pernielsentikaer) [inline extensions/bible/package.json:5]: Can you rephrase this line?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @josmithua 👋 

Thanks for your contribution 🔥 

I have some feedback for you to address: 

- One thing that would be nice to have is when you search for `rev` and the result shows up, if you delete `rev` again it is still present, maybe it should go back to No Results?

- You could add a custom [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) to make the UX better. Something like this (64x64):

CleanShot 2022-04-21 at 11 10 16@2x

And you can change the text accordingly to `No Results` if user input text in search.

File: extensions/bible/package.json (hunk 1 above)
Comment (josmithua) [reply extensions/bible/package.json]: Description updated. I just described the main function of the extension (search) for now. I will update it when I add more features

**Comment by pernielsentikaer** [comment]
Hi @josmithua 👋 

The latest version seems to have broken the "Press Enter To Search", the emptyview needs a tweak to not be triggered with this on, and I get no results at all no matter what I search for

**Comment by josmithua** [comment]
@pernielsentikaer Thanks, silly me forgot to test that again. Fixed now!

**Review by pernielsentikaer — APPROVED** [review]
Hi @josmithua 👋 

Thanks for the corrections. Let's get the first version live 🚀
